addons/survey/data/survey.user_input.line.csv

```diff
@@ -1,4 +1,4 @@
-id,user_input_id:id,question_id:id,skipped,answer_type,value_text,value_number,value_date,value_free_text,value_suggested:id,value_suggested_row:id
```

Comment:
_`englishbitchingmodeon`_

For commit message: "[REF] survey: on survey.user_input.line, rename value_suggested{_row}…"

> A new naming recently introduced for ``survey.question.answer`` has been
> recently introduced

that's a lot of "recently introduced"

> it better indicate the use;

indicates

_`englishbitchingmodeoff`_